warp/yul/utils.py

```diff
@@ -197,4 +197,6 @@ def make_abi_StarkNet_encodable(abi):
     """
     abiStr = f"{abi}".replace("'address'", "'uint256'")
     abiStr = abiStr.replace("'", '"')
+    abiStr = abiStr.replace("True", "true")
+    abiStr = abiStr.replace("False", "false")
```

Comment:
Why do we do that? Also I'm worried about it changing function names like `isTrue` to `istrue`, which would change the selector again

Reply:
oh yeah, I'll fix it now